spec/spec_helper.rb

```diff
@@ -1,8 +1,22 @@
 ENV['RAILS_ENV'] ||= 'test'
 
 require 'simplecov'
+require 'vcr'
```

Comment:
Would be great to remove the VCR dependency, as we are not using it anywhere else and most of the time it is just an overkill and difficult to manage.
In this case I can see there are just a couple fixtures, maybe 3 needed. We can save them as fixtures and in the tests use Webmock to mock the requests and responses.

Reply:
+1